sonic-net/sonic-buildimage
# [Nokia][sonic-platform] Update nokia sonic-platform submodule with thermal coherence changes to be paired with NDK >= 22.9.26

These changes ensure proper thermal subsystem operation when removing ASIC internal thermal sensor polling from NDK.
     - Please pair this submodule update with NDK >= 22.9.26 : NDK 22.9.26 PR is here:  https://github.com/Azure/sonic-buildimage-msft/pull/316

Note that the other non-chassis platform commits included here are correct and serve to synchronize all supported Nokia platforms with sonic-net master image going forward



#### Why I did it
These changes are required in order to ensure thermal subsystem coherence when removing 'bcmcmd show pvt' ASIC thermal sensor polling from NDK.

Please pair this submodule update with NDK >= 22.9.26 : NDK 22.9.26 PR is here:  https://github.com/Azure/sonic-buildimage-msft/pull/316

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Made the necessary modifications to facilitate coherent temp sensor operations.

#### How to verify it
Verify that thermal subsystem operates normally via 'show platform temperature' command.

You will also need separate 202205 branch PR, which is here:  https://github.com/sonic-net/sonic-buildimage/pull/18565

Please pair this submodule update with NDK >= 22.9.26 : NDK 22.9.26 PR is here:  https://github.com/Azure/sonic-buildimage-msft/pull/316



#### Which release branch to backport (provide reason below if selected)




You will also need separate 202205 branch PR, which is here:  https://github.com/sonic-net/sonic-buildimage/pull/18565

#### Tested branch (Please provide the tested image version)




#### Description for the changelog
[Nokia][sonic-platform] Update nokia sonic-platform submodule with thermal coherence changes to be paired with NDK >= 22.9.26



#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
